site/src/pages/news/20200703-newsletter-1.mdx

```diff
@@ -69,11 +69,11 @@ Sign up for our newsletters to keep up-to-date with new releases and useful tips
 <ThankYou
   message={() => <p>Since our last newsletter, we received awesome contributions from the following contributors:</p>}
   usernames={[
-    'adriancole',
     'amitvc',
     'andrey-tpt',
     'anuraaga',
     'cbornet',
+    'codefromthecrypt',
```

Comment:
@codefromthecrypt you should have picked your new name to start with an `a` :p

Reply:
🤣